src/options/coverage.rs

```diff
@@ -0,0 +1,28 @@
+use crate::{options::BuildOptions, project::FuzzProject, RunCommand};
+use anyhow::Result;
+use structopt::StructOpt;
+
+#[derive(Clone, Debug, StructOpt)]
+pub struct Coverage {
+    #[structopt(flatten)]
+    pub build: BuildOptions,
+
+    #[structopt(required(true))]
```

Comment:
Similarly, I think that non-`Option`/non-`Vec` fields are required by default, so this `required(true)` annotation shouldn't be necessary here.

Reply:
Ah, OK. Could you help me understand how this case is different from similar instances in options for the other command? For example, [here](https://github.com/rust-fuzz/cargo-fuzz/blob/master/src/options/run.rs#L10).

Reply:
Unless I'm mistaken, those uses of `required(true)` can also be removed.